### DISCUSSION TITLE
improvement

### DISCUSSION THREAD

**Discussion by dtueditor** [community/community#28129] [Other Feature Feedback, Questions, & Ideas] [answered]
![1](https://user-images.githubusercontent.com/18210674/182511524-cba6174d-9327-498b-be82-2dc2f6c09e29.PNG)
![2](https://user-images.githubusercontent.com/18210674/182511543-1b3b4b47-fa1c-4207-9821-950ef0b7de96.PNG)
![3](https://user-images.githubusercontent.com/18210674/182511547-0aeb3194-ae76-4f79-b7e3-6a0db89b0aba.PNG)

**Comment by jfullstackdev** [comment, marked as answer]
what is it that you are trying to achieve ?

**Reply by dtueditor** [reply]
![1](https://user-images.githubusercontent.com/18210674/182608488-71db3e66-f10a-4015-8ef6-bf6f1d8b8895.PNG)
![2](https://user-images.githubusercontent.com/18210674/182608506-a65f9398-c1bc-47c1-b3d0-0ec92e20e107.PNG)

**Comment by dtueditor** [comment]
come to fruition Bootstrap Menu

**Reply by dtueditor** [reply]
Generally speaking, the container for maximizing a table is not a window, but its parent container. Because there are other elements in the window, the maximization should be completed with the size of the parent container.

**Reply by dtueditor** [reply]
In my improvement, the upper part of the window is the system menu, the lower part is the table toolbar, and the table is maximized in the middle.

**Reply by dtueditor** [reply]
Editing table rows no longer uses dialog boxes, but uses a full screen with adaptive device size. Flow layout of form items.

**Reply by dtueditor** [reply]
JqGrid performs well in desktop systems. To continue its life, it must perform well on mobile phones.

**Reply by dtueditor** [reply]
good luck!

**Comment by dtueditor** [comment]
bootstrap-4.6.2-examples Checkout form use to editGridRow

**Reply by dtueditor** [reply]
use div col-md-6 col-xl-4 mb-3 create form items, table No longer used. it will Adaptive device

**Reply by dtueditor** [reply]
I use easyui create form 
editfrm.panel({
                                    closed: false,
                                    inline: true,
                                    fit: true,
                                    openAnimation: "fade"
                                })

**Comment by jfullstackdev** [comment]
you can also have a short welcome to yourself at the official thread, if you are not yet doing it, here

https://github.com/orgs/community/discussions/23204
